src/components/ui/SearchSort.tsx

```
@@ -0,0 +1,103 @@
+            ? "border-2 border-primary bg-primary-10 text-muted-foreground focus:outline-none"
+            : isOpen
+              ? "border-2 border-primary bg-white text-muted-foreground focus:outline-none"
+              : "border border-[hsl(var(--border))] bg-white text-muted-foreground hover:border-neutral-20 focus:border-transparent focus:outline-none focus:ring-2 focus:ring-primary"
+        }`}
+      >
+        <svg
+          className="h-4 w-4"
+          viewBox="0 0 16 16"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)